.github/workflows/release-build-push.yaml

```diff
@@ -0,0 +1,52 @@
+name: Push Release Image Build
+on:
+  push:
+    tags:
```

Comment:
Could this action also be combined with the other one, and have the script implement any differences if it's a tag?

Reply:
done.